CHANGELOG.md

```diff
@@ -73,7 +73,7 @@ While this is unlikely to be a breaking change in most places, any code that rel
 ##### Other Changes
 
 *  Adjust 2.1 start height ([2e1e44c8](https://github.com/hirosystems/clarinet/commit/2e1e44c8)
-*  Set Clarity2 and epoch 2.1 as default ([6cb1e21d](https://github.com/hirosystems/clarinet/commit/6cb1e21d)
+*  Set Clarity2 and epoch 2.4 as default ([6cb1e21d](https://github.com/hirosystems/clarinet/commit/6cb1e21d)
```

Comment:
Past change logs shouldn't be updated.

epoch 2.4 was used as default in 1.7.1